### ISSUE TITLE
ScrollFrame seems to have a limit height in the patch 9.0.2.35938

### ISSUE THREAD

**Issue by siweia** [Stanzilla/WoWUIBugs#40]
Because blizzard prevent player from opening GuildFrame in combat, I [create a frame](https://github.com/siweia/NDui/blob/Beta/Interface/AddOns/NDui/Modules/Infobar/Guild.lua) that display guild members with copy name and invite unit function, as a replacement of GameTooltip.

In the newest beta patch, once the online members turn more than 97, the scrollframe become not scroll-able.

**Comment by Meorawr** [comment]
Unable to reproduce with this snippet that shoves 10,000 rows into a scrollframe.

```lua
local NUM_ROWS = 10000;

local scrollframe = CreateFrame("ScrollFrame", nil, UIParent, "UIPanelScrollFrameTemplate");
scrollframe:SetPoint("CENTER");
scrollframe:SetSize(315, 320);

local scrollchild = CreateFrame("Frame", nil, scrollframe);
scrollchild:SetSize(315, 1);
scrollframe:SetScrollChild(scrollchild);

for i = 1, NUM_ROWS do
   local widget = CreateFrame("Frame", nil, scrollchild);
   widget:SetPoint("TOPLEFT", 0, -21 * (i - 1));
   widget:SetSize(315, 20);
   
   local background = widget:CreateTexture(nil, "BACKGROUND");
   background:SetAllPoints(true);
   background:SetColorTexture(i / NUM_ROWS, 0, 0);
end
```

**Comment by Nevcairiel** [comment]
Apparently chaining a lot of items in a manner where every next element is anchored to the previous one can cause anchoring issues, while if you SetPoint them all to the same parent with varying offsets its fine.

**Comment by Meorawr** [comment]
That does indeed break it. I threw in an UpdateScrollChildRect after the loop for good measure and it does interestingly seem to be calculating the correct height, it just fails to actually scroll the contents. It fails precisely at 98 rows irrespective of the individual height of the elements.

Updated the comment with a working test case.

**Comment by siweia** [comment]
I thought the 'limit' is intended, and I rewrite the infobar by using HybridScrollFrameTemplate, only create the visible bars and update contents while scrolling, instead of actually create all of the bars. 😓

**Comment by EKE00372** [comment]
change get rollback/fixed?

i didnt fix this issue in my addon  last week, but error gone in this patch.

**Comment by Meorawr** [comment]
Fixed at of both at least 9.0.1.36036 and 9.0.2.36037.
